js/scenes/FTU.js

```diff
@@ -0,0 +1,153 @@
+import React, { Component, PropTypes } from 'react';
+import {
+  View,
+  Image,
+  Text,
+  Button,
+  Dimensions,
+  StyleSheet,
+  Platform,
+} from 'react-native';
+import ViewPager from 'react-native-viewpager';
+import ViewPagerIndicator from '../components/ViewPagerIndicator';
+import { connect } from 'react-redux';
+import { theme } from '../../config';
+
+export class FTU extends Component {
+  constructor(props) {
+    super(props);
+
+    this.navigator = this.props.navigator;
+
+    const PAGES = [
+      () => this.renderPage1(),
+      () => this.renderPage2(),
+    ];
+    const dataSource = new ViewPager.DataSource({
+      pageHasChanged: (p1, p2) => p1 !== p2,
+    });
+
+    this.state = {
+      dataSource: dataSource.cloneWithPages(PAGES),
+    };
+  }
+
+  render() {
+    return (
+      <Image
+        source={require('../images/ftu/background.jpg')}
+        resizeMode="cover"
+        style={styles.container}>
+        <View style={styles.scrim}>
```

Comment:
> *n.* A transparent fabric used as a drop in the theatre to create special effects of lights or atmosphere.

That's a word that came up when I was working on the settings drawer of Firefox OS. I think @djabberwocky was working on the spec back then so he may know more about this term.
That may not be appropriate in this case.